### ISSUE TITLE
Consultation on experimental results

### ISSUE THREAD

**Issue by laoyeeeee** [sunilkmaurya/GNN_Ranking#4]
Hello, I tried to reproduce your experiment. I didn't change any code related to the experiment and just ran the command `python betweenness.py --g SF` directly. But my results are not good, the average KT-SCORE is around 0.25. Is there something I'm missing?

**Comment by sunilkmaurya** [comment]
Hi,
Thanks for your interest in our work.
The code was written for very old PyTorch version (0.4). Since then, there have been some breaking changes in torch and other packages. One potential issue can be that if you are working with newer PyTorch version, you would need to add `self.training` in `F.dropout()`, otherwise dropout does not work as intended, and causes lower performance. There is a related comment in the Readme.

**Comment by laoyeeeee** [comment]
Thanks for your reply. 
I have changed the code by your instruction actually. But my results still work poor. I'd appreciate it if you can give me more suggestions.

**Comment by sunilkmaurya** [comment]
It is hard to say what could be the problem. New packages may have some modified implementation or deprecated some functionalities. I quickly set up a virtual environment with a bit older versions of some packages. The command seems to have similar to expected output. Please have a look at the attached `environment.txt` file to check package versions. Hoping it helps a bit. 

[output.txt](https://github.com/user-attachments/files/19381770/output.txt)

[environment.txt](https://github.com/user-attachments/files/19381744/environment.txt)
